### DISCUSSION TITLE
Release 5.1.0 Deployment issues

### DISCUSSION THREAD

**Discussion by sandeep-r1** [Sunbird-Ed/Community#359] [Bugs] [answered]
**1)Nginx-public-ingress Deployment Issue**
We are trying to deploy Deploy/Kubernetes/nginx-public-ingress with public tag release-5.1.0_RC5 and facing the following error in jenkins. (jenkins job link attached in the weblinks)

 

`TASK [helm-deploy : rename template] *******************************************
fatal: [localhost]: FAILED! => {
    "changed": false
}

MSG:

AnsibleUndefinedVariable: list object has no element 2

PLAY RECAP *********************************************************************
localhost                  : ok=2    changed=1    unreachable=0    failed=1    skipped=1    rescued=0 `

2)Content service Deployment.
Content Editor deployment is job is failed with below error:

Tag: release-5.1.0_RC5
`TASK [azure-cloud-storage : delete files and folders from azure storage using azcopy] ***
fatal: [localhost]: FAILED! => {
    "ansible_job_id": "337217359807.5713",
    "changed": true,
    "cmd": "azcopy rm 'https://preprodall.blob.core.windows.net/sunbird-content-preprod/content-editor?se=2023-01-18T11%3A38Z&sp=rwdl&spr=https&sv=2018-11-09&sr=c&sig=lviojJgGMxgC30suWTvQ5ft94Qi61LjlR/2nIVJfH/I%3D' --recursive",
    "delta": "0:00:00.172462",
    "end": "2023-01-18 16:08:25.250392",
    "finished": 1,
    "rc": 1,
    "start": "2023-01-18 16:08:25.077930"
}

STDOUT:

INFO: Any empty folders will not be processed, because source and/or destination doesn't have full folder support

failed to perform remove command due to error: cannot start job due to error: cannot list files due to reason -> github.com/Azure/azure-storage-blob-go/azblob.newStorageError, /home/vsts/go/pkg/mod/github.com/!azure/azure-storage-blob-go@v0.10.1-0.20210407023846-16cf969ec1c3/azblob/zc_storage_error.go:42
===== RESPONSE ERROR (ServiceCode=ContainerNotFound) =====`

**Comment by Krishnaj20** [comment]
@vinukumar-vs @gandham-santhosh - pls check and respond

**Reply by gandham-santhosh** [reply, marked as answer]
@sandeep-r1 I see that script is looking for sunbird-content-preprod container and could not perform operation as it is not present. Please check the container name you already have and update the value in private repo common.yaml accordingly

**Reply by chitranshu-keshav** [reply]
@sandeep-r1 Can you please mark this thread as answered if your query is resolved?
